contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -984,3 +984,28 @@ def perform_create(self, serializer, change=None):
                 created_by_id=change["created_by_id"],
                 applied=True
             )
+
+    def update_descendants(self, pk, mods):
+        """ Update a node and all of its descendants with the given mods """
+        root = ContentNode.objects.get(id=pk)
+
+        if root.kind_id != content_kinds.TOPIC:
+            raise ValidationError("Only topics can have descendants to update")
+
+        descendants = root.get_descendants(include_self=True).values("id")
```

Comment:
Could use `values_list` here to save Django from creating the dictionaries just to hold one value-- the id.

Reply:
Changed!